presto-cli/src/main/java/com/facebook/presto/cli/QueryRunner.java

```diff
@@ -68,7 +68,7 @@ public QueryRunner(
 
         OkHttpClient.Builder builder = new OkHttpClient.Builder();
 
-        setupTimeouts(builder, 5, SECONDS);
+        setupTimeouts(builder, 30, SECONDS);
```

Comment:
Yes, it's going to affect both. Commit message just gives v2 as an explanation why. I think it shouldn't be a problem to increase it for both. However, if someone thinks otherwise I can configure different timeouts based on protocol version.